src/components/Tooltip.svelte

```diff
@@ -37,33 +50,31 @@
     const bottom = parentRef.children[0]
       ? parentRef.children[0].offsetHeight
       : 0;
+    let newInset;
 
-    if (left < 0) {
-      return {
-        ...tooltipInset,
-        left: 0,
-        translateX: 0,
-        bottom
-      };
-    } else if (right > window.innerWidth) {
-      return {
-        right: 0,
-        left: 'auto',
-        translateX: 0,
-        bottom
-      };
-    } else {
-      return { ...defaultTooltipInset, bottom };
+    switch (true) {
```

Comment:
honestly saying `switch (true)` is also not the best way to do that )
have you saw lodash cond? Maybe it could help?
https://lodash.com/docs/4.17.15#cond

Reply:
Seems cool